support/lobby/lib/utils.js

```diff
@@ -0,0 +1,34 @@
+export function get( resource ) {
+  return fetch( resource, getOptions() ).
+    then( responseBody );
+}
+
+export function post( resource, body ) {
+  return fetch( resource, postOptions( body ) ).
+    then( responseBody );
+}
+
+function getOptions() {
+  return {
+    credentials: "same-origin",
+  };
+}
+
+function postOptions( body ) {
+  let json = typeof body === "object" && body !== null &&
```

Comment:
I wouldn't mind some parentheses here to visually delimit the conditions from the conjunctions.